src/components/report-viewer-tab.js

```diff
@@ -104,21 +119,32 @@ export const ReportViewerTab = ({ studyId, visible, workingNode }) => {
         <WaitingLoader loading={waitingLoadReport} message={'loadingReport'}>
             {report && (
                 <Paper className={clsx('singlestretch-child')}>
-                    <FormControlLabel
-                        className={classes.reportOnlyNode}
-                        control={
-                            <Switch
-                                checked={nodeOnlyReport}
-                                inputProps={{
-                                    'aria-label': 'primary checkbox',
-                                }}
-                                onChange={(e) => handleChangeValue(e)}
-                            />
-                        }
-                        label={intl.formatMessage({
-                            id: 'LogOnlySingleNode',
-                        })}
-                    />
+                    <div className={classes.div}>
+                        <FormControlLabel
+                            className={classes.reportOnlyNode}
+                            control={
+                                <Switch
+                                    checked={nodeOnlyReport}
+                                    inputProps={{
+                                        'aria-label': 'primary checkbox',
+                                    }}
+                                    onChange={(e) => handleChangeValue(e)}
+                                />
+                            }
+                            label={intl.formatMessage({
+                                id: 'LogOnlySingleNode',
+                            })}
+                        />
+                        {!allowModificationsOnNode(workingNode, selectedNode) &&
+                            selectedNode?.type !== 'ROOT' && (
```

Comment:
Same as before, maybe use a function whose name makes sense with the warning displayed.

Reply:
Done